src/util/addBoxPlot.js

```diff
@@ -1,107 +1,151 @@
-export default function addBoxplot(svg, results, height, width, domain, boxPlotWidth, boxColor, boxInsideColor, format, horizontal){
```

Comment:
Would definitely be good to share these improvements. At the very least we should put them in a new stand-alone example so that we don't forget about them. 

Still thinking it would be nice to implement a boxplot 'mark' in webcharts. Maybe a worthy Rho24?